admin/helpers/hdf-presenter.js

```diff
@@ -11,4 +11,26 @@ hdfPresenter.getCanViewResults = (resultsDate) => {
   return moment().utc().isAfter(resultsDate)
 }
 
+hdfPresenter.getPupilsWithProcessStatus = (pupils) => {
```

Comment:
This isn't a full process as it does not include "Restart"... so what is it?

Reply:
@jon-shipley changed the naming to getPupilsWithViewStatus as it concerns a specific number of statuses that are eligible for display(Incomplete, complete, actual reason for not taking the check).